wcomponents-core/src/main/java/com/github/bordertech/wcomponents/WebUtilities.java

```diff
@@ -545,11 +552,21 @@ public static void updateBeanValue(final WComponent component)
             // Update the rest of the bean
             for (int i = ((Container) component).getChildCount() - 1; i >= 0; i--)
             {
-                updateBeanValue(((Container) component).getChildAt(i));
+                updateBeanValue(((Container) component).getChildAt(i), ignoreVisible);
             }
         }
     }
-
+    
+        /**
+     * Updates the bean value with the current value of the component
+     * and all its bean-bound children. By default this method will process invisible containers and inputs.
+     *
+     * @param component the component whose contents need to be copied to the bean.
+     */
+    public static void updateBeanValue(final WComponent component)
+    {

```

Comment:
If the flag changes to "visible only", then this will change to default "true". As this will have a possible impact to projects as they might have been getting some unexpected behaviour (i.e. the invisible components being updated). This will need to be identified as an impact in the change log.